zerver/lib/user_groups.py

```diff
@@ -899,6 +902,13 @@ def create_system_user_groups_for_realm(realm: Realm) -> dict[str, NamedUserGrou
         everyone_on_internet_group_info,
     ]
 
+    from zerver.actions.user_groups import render_group_description
```

Comment:
Can we just define `render_group_description` in this file itself, ie. `zerver/lib/user_groups.py` if that doesn't result in any import cycles?

Reply:
Ok, this seems fine for now then.